wordpress-mobile/WordPress-Android
# [Blogging prompts v1] Pass prompt ID instead of prompt content to editor screen

Fixes #16501 

To test:
1 - Onboarding 
- Open Debug settings and enable `blogging_prompts_notification_remote_field`
- Open app logged in and with a site selected
- Click on the blogging prompts onboarding notification OR in the "Learn more" button
- Blogging prompts onboarding dialog should be seen
- Tap on "Try it Now" button
- Prompt ID should be passed to editor screen

2 - Reminder notification
- Open Debug settings and enable `blogging_prompts_notification_remote_field`
- On `My Site`, tap on `Menu`
- Tap on `Site Settings`
- Tap on `Reminders and prompts` and select a date and time for the reminder
- Change the device date to something after the selected date/time for the reminder
- A notification to answer the prompt should appear
- Tap on the notification
- Prompt ID should be passed to editor screen

3 - My Site card
- Open Debug settings and enable `blogging_prompts_remote_field`
- On `My Site`, tap on `Home`
- Scroll down until you see the prompts card (with the text `Cast the movie of your life`)
- Tap on `"Answer prompt"` 
- Prompt ID should be passed to editor screen

4 - Compact card
- Open Debug settings and enable `blogging_prompts_remote_field`
- On `My Site`, tap on the FAB. You should see the option to answer prompt.
- Tap on `"Answer prompt"`
- Prompt ID should be passed to editor screen


## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
Updated existing unit tests

PR submission checklist:
